ucsb_ds_capstone_projects_2021/projects/ccber/update1.md

```diff
@@ -17,24 +26,41 @@ Then, we discussed our results from using rglobi, the GloBI Web API, and Cypher.
 
 The goals after this meeting were to take some time to explore the raw dataset, since it was huge, check for reflexiveness in the data, and begin thinking about how we want to visualize these pollination networks. Additionally, we needed to brainstorm how to work with large data files that need to be shared and updated periodically. Some of us chose to explore the dataset with R while others were more comfortable with Python.
 
-## Week 3
+### Week 3
 We discussed the attributes and qualities of the raw data. Also, we explored different options for querying and subsetting the data in more detail. Our sponsors provided more insight on variations in the interactions data. Some of this variation includes: size, coloration, foraging behavior, and sociality of bees. There is significant diversity in the Apidae bee family. Classification and taxonomy can be used to filter some of this variation. 
 
 ![](https://github.com/angelchen7/ucsb-ds-capstone-2021.github.io/blob/main/ucsb_ds_capstone_projects_2021/projects/ccber/different_bees.png?raw=true "variation in Apidae")
```

Comment:
Same here
Markdown can embed images directly from the same directory: https://jupyterbook.org/content/figures.html
For the image  `some-image.png` in the same directory (you already have it), replace the HTML with `![some-text](`some-image.png)`